cpp/dolfinx/io/xdmf_utils.cpp

```diff
@@ -575,10 +583,12 @@ xdmf_utils::distribute_entity_data(const mesh::Mesh& mesh, int entity_dim,
       send_nodes_owned[p1].insert(send_nodes_owned[p1].end(), e_recv.begin(),
                                   e_recv.end());
       send_vals_owned[p1].push_back(_data_recv[e]);
+      ++total_size;
     }
   }
 
   // TODO: Pack into one MPI call
+  LOG(INFO) << "XDMF return entity and value data (" << total_size << ")";
```

Comment:
Do we nee to add a size? Counter is a bit ugly, and the log message doesn't say what the number is.

Reply:
size is useful to see where we are using lots of memory (which is the main issue). I'll try to do it without a counter.